src/parsers.rs

```diff
@@ -152,7 +152,12 @@ named!(pub parse_bit<bool>, alt!(
 
 /// Parses a usize followed by a kB unit tag.
 named!(pub parse_kb<usize>,
-       chain!(space ~ bytes: parse_usize ~ space ~ tag!("kB"), || { bytes }));
+       do_parse!(
+           space >>
+           bytes: parse_usize >>
+           space >>
+           opt!(tag!("kB")) >>
```

Comment:
I may have misinterpreted the logic of the chain, this may not need to be wrapped in `opt!`?